buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -171,7 +171,8 @@ private object Libraries {
 
     // Accompanist
     // https://github.com/google/accompanist/releases
-    const val ACCOMPANIST = "com.google.accompanist:accompanist-webview:0.24.5-alpha"
+    const val ACCOMPANIST_WEBVIEW = "com.google.accompanist:accompanist-webview:0.24.5-alpha"
```

Comment:
As there are now multiple dependencies relying on the same version number, go ahead and create a const val holding the version and replace all hardcoded version numbers with that property reference.

It's nice to have a single source of truth to update when new dependency versions are available.

```
private const val ACCOMPANIST_VERSION = "0.24.5-alpha"
```

Reply:
will do